config/routes.rb

```diff
@@ -270,6 +270,7 @@
     constraints(Constraints::Admin) do
       namespace :admin, constraints: Constraints::Admin::RubygemsOrgAdmin do
         mount GoodJob::Engine, at: 'good_job'
+        mount MaintenanceTasks::Engine, at: "maintenance_tasks"
```

Comment:
nit: Can add a button on `http://localhost:3000/admin/resources/maintenance_tasks_runs` to the maintenance_tasks dashboard for easier access

Reply:
There's no easy way to add any component to an avo index page as far as I can tell